# cd /data/openpilot && curl https://raw.githubusercontent.com/eFiniLan/openpilot-patches/master/0.5.10/toyota/for_stock_acc_only_does_not_disable_op_on_gas.diff | git apply -v
#
# (Suitable for Stock ACC)
# This patch will disable OP disengagement when gas is pressed.
#

--- selfdrive/car/toyota/interface.py
+++ selfdrive/car/toyota/interface.py
@@ -346,13 +346,9 @@
       events.append(create_event('pcmDisable', [ET.USER_DISABLE]))
 
     # disable on pedals rising edge or when brake is pressed and speed isn't zero
-    if (ret.gasPressed and not self.gas_pressed_prev) or \
-       (ret.brakePressed and (not self.brake_pressed_prev or ret.vEgo > 0.001)):
+    if ret.brakePressed and (not self.brake_pressed_prev or ret.vEgo > 0.001):
       events.append(create_event('pedalPressed', [ET.NO_ENTRY, ET.USER_DISABLE]))
 
-    if ret.gasPressed:
-      events.append(create_event('pedalPressed', [ET.PRE_ENABLE]))
-
     ret.events = events
     ret.canMonoTimes = canMonoTimes